internal/v4/relations.go

```diff
@@ -264,6 +264,16 @@ func (h *ReqHandler) metaBundlesContaining(entity *mongodoc.Entity, id *router.R
 	includes := flags["include"]
 	// TODO(rog) make this concurrent.
 	for _, e := range entities {
+		rurl := router.ResolvedURL{
+			URL:                 *e.URL,
+			PromulgatedRevision: -1,
+		}
+		if e.PromulgatedURL != nil {
+			rurl.PromulgatedRevision = e.PromulgatedURL.Revision
+		}
+		if err := h.AuthorizeEntity(&rurl, req); err != nil {
```

Comment:
Perhaps add a comment:
// Ignore entities for which the user does not have permission.

?